{"uuid":"85f25114-f501-4341-9fab-4527bff716d1","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["5c058d66-265d-4dc7-90d7-de5b19b22118"],"befores":[],"afters":[{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696471076057,"stop":1696471076058},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696471076150,"stop":1696471076152},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696471076198,"stop":1696471076199},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696471078223,"stop":1696471078223}],"start":1696471075873,"stop":1696471078223}